docs/data/system/getting-started/usage/Demo.js

```diff
@@ -30,38 +29,42 @@ export default function Demo() {
       />
       <Box
         sx={{
+          p: 3,
+          minWidth: { md: 350 },
           display: 'flex',
           flexDirection: 'column',
           alignItems: { xs: 'center', md: 'flex-start' },
-          m: 3,
-          minWidth: { md: 350 },
+          gap: 0.5,
         }}
       >
-        <Box component="span" sx={{ fontSize: 16, mt: 1 }}>
+        <Box component="span" sx={{ fontSize: 14, color: 'text.secondary' }}>
           123 Main St, Phoenix AZ
         </Box>
-        <Box component="span" sx={{ color: 'primary.main', fontSize: 22 }}>
+        <Box
+          component="span"
+          sx={{ color: 'primary.main', fontSize: 24, fontWeight: 'bold' }}
+        >
           $280,000 — $310,000
         </Box>
         <Box
           sx={{
-            mt: 1.5,
-            p: 0.5,
-            backgroundColor: (theme) => alpha(theme.palette.primary.main, 0.1),
-            borderRadius: '5px',
-            color: 'primary.main',
-            fontWeight: 'medium',
+            py: 0.5,
+            px: 1,
+            backgroundColor: (theme) => alpha(theme.palette.success.dark, 0.1),
```

Comment:
This is where I got the error when running the regression test locally.

Reply:
Any idea why that's failing? I don't see anything when running the PR locally... pushed a commit changing it to use `main` instead, but I'd expect it to work for both `light` and `dark`... 🤔

Reply:
Seems not to have worked as well — this demo was already using the theme function there, so I wonder why it's not working anymore once if flipped to use the System package instead...

Reply:
System package's theme doesn't have tokens defined. Seems to be the reason. Box from material comes with the material theme.